lte/gateway/python/scripts/generate_oai_config.py

```diff
@@ -331,6 +333,44 @@ def _get_amf_name_config(service_mconfig: object) -> str:
     return service_mconfig.amf_name or DEFAULT_NGAP_AMF_NAME
 
 
+def _get_default_auth_retry_count(service_mconfig: object) -> str:
+    """Retrieve default_auth_retry_count config value. If it does not exist, it defaults to DEFAULT_AUTH_RETRY_COUNT.
+
+    Args:
+        service_mconfig: This is a configuration placeholder for mme.
+
+    Returns:
+        default default auth retry count.
+    """
+    enable_default_auth_retry_count_config = get_service_config_value(
+        'mme', 'auth_retry_max_count', None,
+    )
+
+    if enable_default_auth_retry_count_config is not None:
+        return enable_default_auth_retry_count_config
+
+    return DEFAULT_AUTH_RETRY_COUNT
+
+
+def _get_default_auth_timer_expire_msec(service_mconfig: object) -> str:
+    """Retrieve default_auth_retry_timer_expire_msec config value. If it does not exist, it defaults to DEFAULT_AUTH_TIMER_EXPIRE_MSEC.
```

Comment:
same here, indentation looks like it's not done

Reply:
comment addressed.